### ISSUE TITLE
Configure post-build copy location

### ISSUE THREAD

**Issue by DebugXYZ** [rei/febs#35]
Sometime after the front-end build the contents of ./dist may need to be copied elsewhere. There should be a facility for this as well as a way to specify the location in which to copy to.

To be clear, febs should always build to `./dist`, but after it does it can be copied elsewhere by other mechanisms.

This could be dependent on issue #27

**Comment by peripateticus** [comment]
This begs the question: Instead of needing to add more copy to copy from an unchangeable `/dist` directory, why not simply allow the client to set the output directory as needed by their application.

**Comment by DebugXYZ** [comment]
Good point, we may be able to remove this requirement with some work. We may consider adding this is a configuration in #27

**Comment by DebugXYZ** [comment]
We might even consider adding as a build flag if anybody would think thats useful

**Comment by DebugXYZ** [comment]
I believe the need for this has been resolved in #27
